### PR TITLE
PMM-5097 Add client docker to be added and tagged

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Comment by AlekSi** [comment]
Not for 2.2.1.

**Comment by AlekSi** [comment]
@hors please review it, but do not merge.

File: pmm/pmm2-server-release.groovy [hunk 1]
```diff
@@ -177,17 +177,24 @@ pipeline {
                         fi
                         docker save percona/pmm-server:\${VERSION} | xz > pmm-server-\${VERSION}.docker
 
-                        #docker pull \${DOCKER_CLIENT_VERSION}
-                        #docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
-                        #docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
-                        #docker push perconalab/pmm-client:\${VERSION}
-                        #docker save perconalab/pmm-client:\${VERSION} | xz > pmm-client-\${VERSION}.docker
+                        docker pull \${DOCKER_CLIENT_VERSION}
```
Comment (hors) [inline pmm/pmm2-server-release.groovy]: We do not have DOCKER_CLIENT_VERSION option  for this job. Please add it.

Comment (EvgeniyPatlan) [reply pmm/pmm2-server-release.groovy]: done

File: pmm/pmm2-server-release.groovy [hunk 2]
```diff
@@ -177,17 +181,24 @@ pipeline {
...
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${DOCKER_MID}
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${TOP_VER}
+                        if [ \${TOP_VER} = 1 ]; then
```
Comment (AlekSi) [inline pmm/pmm2-server-release.groovy]: @EvgeniyPatlan @hors shouldn't it be `2`?

Comment (AlekSi) [reply pmm/pmm2-server-release.groovy]: @Sevlin ^

Comment (hors) [reply pmm/pmm2-server-release.groovy]: @AlekSi As I know we do not tag pmm 2 as latest for now.

Comment (AlekSi) [reply pmm/pmm2-server-release.groovy]: True. But we also do not build PMM 1 with that job. So why this check exist at all?

**Pull request merged**